kythe/go/platform/kzip/kzip.go

```diff
@@ -98,40 +98,26 @@ import (
 )
 
 // Type Encoding describes how compilation units will be encoded when written to a kzip.
-type Encoding int32
+type Encoding int
 
 const (
-	BOTH  Encoding = 0
-	JSON  Encoding = 1
-	PROTO Encoding = 2
-)
+	EncodingBoth  Encoding = 0
```

Comment:
"Both" as a name seems a bit problematic and precludes future changes.  I don't think those are likely, but would prefer "All".